src/main/resources/templates/home.html

```diff
@@ -1,15 +1,37 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml"
       xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      xmlns:th="https://www.thymeleaf.org"
       layout:decorate="layout">
     <head>
         <title>Home</title>
+        <link th:href="@{/css/application.css}" rel="stylesheet" />
+        <link th:href="@{/css/home-page-style.css}" rel="stylesheet" />
+
     </head>
     <body>
         <section layout:fragment="content">
-            <h1>Welcome to Whale Spotting!</h1>
+            <div class="grid-container">
+                <div class="grid-item-1"><p>Welcome Here is where we will be putting our introductory information:...It’s possible to watch whales and dolphins without leaving dry land (perfect for those who want to avoid seasickness!).  We recommend  viewing from cliff tops and other vantage points whenever possible as  this reduces the likelihood of whales and dolphins being disturbed.</p></div>
```

Comment:
Add indenting to clean this line up, and clean up the spacing and punctuation
```suggestion
                <div class="grid-item-1">
                    <p>
                        Welcome
                        Here is where we will be putting our introductory information:
                        It’s possible to watch whales and dolphins without leaving dry land (perfect for those who want to avoid seasickness!).
                        We recommend viewing from cliff tops and other vantage points whenever possible as this reduces the likelihood of whales and dolphins being disturbed.
                    </p>
                </div>
```